plugins/shared/info-panel/index.js

```diff
@@ -23,10 +23,12 @@ require("./style.less");
 const INITIAL_PANEL_MARGIN_PX = 10;
 const COLLAPSED_CLASS_NAME = "tota11y-collapsed";
 const HIDDEN_CLASS_NAME = "tota11y-info-hidden";
+const STATUS_PANEL_VIEW_CLASS_NAME = "tota11y-info-status-panel-view";
```

Comment:
We're creating a slightly smaller Panel for use with plugins like this that don't have information to display in the larger panel and making it something that can be toggled by a plugin with an option passed into the `InfoPanel` constructor.

Reply:
That seems reasonable.